src/control/cmd/dmg/pool.go

```diff
@@ -213,7 +213,7 @@ func (cmd *PoolDestroyCmd) Execute(args []string) error {
 	err := control.PoolDestroy(ctx, cmd.ctlInvoker, req)
 
 	if cmd.jsonOutputEnabled() {
```

Comment:
do we need this check?

Reply:
Nope, good catch. I had to push again after getting the fixes for mpiio/romio from master, so I've just removed it.